GetIntoTeachingApi/Jobs/UpsertModelWithCandidateIdJob.cs

```diff
@@ -78,7 +78,7 @@ public void Run(string json, PerformContext context)
             }
 
             var duration = (DateTime.UtcNow - _contextAdapter.GetJobCreatedAt(context)).TotalSeconds;
-            _metrics.HangfireJobQueueDuration.WithLabels(new[] { $"UpsertModelJob<{typeName}>" }).Observe(duration);
+            _metrics.HangfireJobQueueDuration.WithLabels($"UpsertModelJob<{typeName}>").Observe(duration);
```

Comment:
is the `$` required in the argument ?

Reply:
Yeah its for interpolating the `{typeName}` into the string